app.rb

```diff
@@ -1,9 +1,35 @@
 require 'sinatra/base'
+require 'sinatra/reloader' # start auto reload
+require_relative './lib/chittermanager' 
 
 class Chitter < Sinatra::Base
+  register Sinatra::Reloader 
+
   get '/test' do
     'Test page'
   end
 
+  get '/' do
+    erb :index
+  end
+  
+  get '/view' do
```

Comment:
Very nice that you're using the same route for both `.all` and `.filter`.  Not many people figure this one out.